tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/FireOnStartCommon.cs

```diff
@@ -79,12 +79,8 @@ public static class FireOnStartCommon
 
                 if (iastEnabled && IastModule.AddRequestVulnerabilitiesAllowed())
                 {
-                    if (span is not null)
-                    {
-                        var statusCode = instance.DuckCast<HttpProtocolStruct>().StatusCode;
-                        ReturnedHeadersAnalyzer.Analyze(responseHeaders, IntegrationId.AspNetCore, span.ServiceName, statusCode);
-                    }
-
```

Comment:
Is this the change that meant the span IDs became stable?

Reply:
No, it was the amount of spans expected in the tests 🙄